drivers/sensor/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+ccflags-y +=-I$(srctree)/drivers
+
+zephyr_sources_ifdef(CONFIG_ADXL362 adxl362/
```

Comment:
Missing parenthesis here.

Reply:
And something looks off. It's adding a source, but using a directory as an argument.